src/pages/components/checkbox.mdx

```diff
@@ -1,15 +1,183 @@
 ---
 title: Checkbox
 description: >
-  Description will be here
+  A Checkbox lets a user select an option. It provides an easy way to make a choice, and helps
+  users compare options.
+package: '@zendeskgarden/react-forms'
+components:
+  - forms/src/elements/Checkbox.tsx
+  - forms/src/elements/common/Field.tsx
+  - forms/src/elements/common/Hint.tsx
+  - forms/src/elements/common/Label.tsx
+  - forms/src/elements/common/Message.tsx
 ---
 
-Content here
+<Usage>
+  <Use>
+    <ul>
+      <li>To select any number of options, including zero</li>
+      <li>As a standalone control to turn a single option on or off</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        To limit the user to selecting exactly one option from a set, use{' '}
+        <a href="../radio">Radios</a>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+A Checkbox’s label is part of its touch target.
+
+import Default from '../../examples/forms/checkbox/Default.tsx';
+import DefaultCode from '!!raw-loader!../../examples/forms/checkbox/Default.tsx';
+
+<CodeExample code={DefaultCode}>
+  <Default />
+</CodeExample>
+
+### Indeterminate
+
+If a checkbox has child checkboxes, it should show an indeterminate state (reflected by a dash) when
+the child checkboxes contains both some checked and unchecked checkboxes.
```

Comment:
> child checkboxes contains both some

Is this grammatically correct? Parent-child interactions can mean specific things on the dev side, specifically around DOM hierarchy.

Maybe a more correct wording would be "grouping".

Reply:
> Is this grammatically correct?

I'm not sure, but it sounds like the sentence could be improved a lot. I was hoping @vsri1137 could help check out the example and wordsmith that description 🤞🏽.

> Maybe a more correct wording would be "grouping".

I think the word "grouping" doesn't convey a (visual, not DOM) parent-child relationship. The term "child checkboxes" does convey that visual relationship. Apple HCI guidelines use [similar](https://developer.apple.com/design/human-interface-guidelines/macos/buttons/checkboxes/) terminology. Maybe there is another term that conveys a visual parent-children hierarchy?

Reply:
@austin94 Given the context, is it likely that there would be confusion here around the use of parent-child? 

If so, perhaps something more accurate that grouping (which implies several options that are all at the same level) we could use ancestor-descendant. 

However, if there is little chance that someone reading this will mistake this for DOM usages, then I'd rather we stick with more well-known, less jargon-y words.

Reply:
Proposed:

> If the children (or descendants) of a Checkbox have different states (some on, some off) the parent Checkbox will show an indeterminate state.

Reply:
Thanks V, that sounds great. Huge improvement!